server/chat-plugins/scavenger-games.ts

```diff
@@ -213,6 +219,45 @@ const TWISTS: {[k: string]: Twist} = {
 		},
 	},
 
+	spamfilter: {
+		name: 'Spam Filter',
+		id: 'spamfilter',
+
+		desc: 'Every wrong answer adds 30 seconds to your final time!',
+
+		onIncorrectAnswer(player: ScavengerHuntPlayer, value: string) {
+			if (!player.incorrect) player.incorrect = [];
+			const id = `${player.currentQuestion}-${value}`;
+			if (player.incorrect.includes(id)) return;
+
+			player.incorrect.push(id);
+		},
+
+		onComplete(player, time, blitz) {
+			const seconds = toSeconds(time);
+			if (!player.incorrect) return {name: player.name, total: seconds, blitz, time, original_time: time};
```

Comment:
What if `player.incorrect` is an empty array?

Reply:
Oh, hmm, it seems like this wouldn't affect anything aside from potentially letting `blitz` remain true even when the seconds was greater than 60.